src/lib/http-post-parser.ts

```diff
@@ -205,7 +210,7 @@ export class HttpPostParser {
       -----------------------------1809321333852408290275809649
       Content-Disposition: form-data; name="email"
       [snipped]
-     */
+     * /
```

Comment:
Hmm, interesting, nested comment blocks :D
I wouldn't be against using single-line-comments for this entire block of "saving this for later" code - seems like it's intended to be uncommented later anyway.

Reply:
That would indeed be clearer, but "saving for later" code should not even exist :) - This is a very temporary code block (it will either be removed or reinstated very soon). Referenced in the code cleanup issue at https://github.com/mozilla/openwpm-webext-instrumentation/issues/21